### PR TITLE
Bug/misc

### PR DESCRIPTION
**Pull request by MRyderOC**
Miscellaneous bug fixes: BUG #110 , BUG #111, BUG #112

### CODE REVIEW

File: src/dfcx_scrapi/core/webhooks.py
Comment (kmaphoenix) [inline src/dfcx_scrapi/core/webhooks.py:1]: I think this method is working as designed. See my comment in the filed bug. We should remove this from the PR unless you can show a repro of this not working for some combination of update kwargs.

**Review by kmaphoenix — CHANGES_REQUESTED** [review]
I think the update_webhook patch is not necessary based on my other comment.  
Everything else LGTM

**Comment by MRyderOC** [comment]
Thanks @kmaphoenix for your review and your time. Per our discussion, changes to the update_webhook have been reverted. So, it's safe to merge for now. Further action needs to be taken upon update_TYPE due to builders.

**Review by kmaphoenix — APPROVED** [review]
LGTM

**Pull request merged**